app/src/main/java/co/kr/soptandroidseminar/util/MyUtil.kt

```diff
@@ -0,0 +1,35 @@
+package co.kr.soptandroidseminar.util
+
+import android.content.Context
+import android.util.Log
+import android.widget.Toast
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+fun Context.simpleToast(message: String) {
+    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
+}
+
+fun <ResponseType> Call<ResponseType>.enqueueUtil(
+    onSuccess: (ResponseType) -> Unit,
+    onError: ((stateCode: Int) -> Unit)? = null
```

Comment:
```suggestion
    onError: ((stateCode: Int) -> Unit) = { _ -> }
```

대애충 이렇게 해서 onError를 null로 안 만들 수도 있음

Reply:
null로 안 만들 수도 있다 ... 그렇다 !